tools/upload_to_aws.py

```diff
@@ -17,40 +21,45 @@
 idx_file_name = task_dict.get('input').get('idx_file_name')
 idx_object_id = task_dict.get('input').get('idx_object_id')
 idx_file_md5sum = task_dict.get('input').get('idx_file_md5sum')
+project_code = task_dict.get('input').get('project_code')
 
-if file_md5sum is None:
-    file_md5sum = str(get_md5(file_))
 
+task_start = int(time.time())
+file_size = 0
+run = False
 
-if idx_file_ and idx_file_md5sum is None:
-    idx_file_md5sum = str(get_md5(idx_file_))
+if project_code in allowed_codes:
+    run = True
+    
+    if file_md5sum is None:
+        file_md5sum = str(get_md5(file_))
 
-file_size = int(os.path.getsize(file_))
+    if idx_file_ and idx_file_md5sum is None:
+        idx_file_md5sum = str(get_md5(idx_file_))
 
+    file_size = int(os.path.getsize(file_))
 
-task_start = int(time.time())
+    if idx_object_id:
+        file_size+= int(os.path.getsize(idx_file_))
```

Comment:
I think it's still good to have two different variables file size, this way we know the precise size for each of the files. For George to get the total size, he simply can perform the addition on his own.